app/assets/src/stylesheets/README.md

```diff
@@ -0,0 +1,25 @@
+# How We CSS
+
+## Modularity
+
+We like the thing Brad Frost calls "[atomic design](http://bradfrost.com/blog/post/atomic-web-design/)," which is (confusingly) practically the opposite of the thing a lot of people call "[atomic CSS](https://acss.io/)." We like "atomic design" in the "pattern library" sense for this project because pattern libraries let people who aren't designers build acceptable and maintainable user interfaces quickly.
+
+We've added a "base" directory to Frost's schema for things like colors, typography, shared functions/mixins, and user-agent-stylesheet normalization. The line between "base" and "atoms" is kinda squishy. Let's play it by ear! In general, let's err on the side of moving stuff into "atoms" if we're not sure if it goes in "base" or "atoms."
+
+## SCSS-specific stuff, in lieu of a linter
+
+DO:
+* Use variables and mixins a lot.
+* Use functions if variables and mixins aren't cutting it.
+
+TRY NOT TO:
+* Nest selectors more than a level or so deep -- do you need the specificity? Is there a way to refactor so that you don't?
```

Comment:
Is this me being too much of a cranky old lady?

Reply:
I mean, if we're going with thing--descriptor I don't see a strong need for deep nesting.